src/main/groovy/edu/wpi/first/gradlerio/wpi/dependencies/WPINativeJsonDepRules.groovy

```diff
@@ -86,7 +86,7 @@ class WPINativeJsonDepRules extends RuleSource {
                                 } else {
                                     lib.staticMatchers.add("**/*${cpp.libName}*.a".toString())
                                 }
-                                lib.maven = "$mavenbase:${platform}${linkSuff}debug@zip"
+                                lib.maven = "$mavenbase:${platform}${linkSuff}$buildKind@zip"

```

Comment:
Should this be `${buildKind}`?

Reply:
It doesn't need to be no, we are very bad at consistency with that. Like that same line already uses both, so I just matched the other version of this for non rio libs.